### PR TITLE
read_era_info() implementation

### PR DESCRIPTION
**Pull request by Maar-io**

### CODE REVIEW

**Comment by Maar-io** [comment]
This contract version breaks during deployment on the Polkadot.js

File: dapps-staking-chain-extension/lib.rs [hunk 1]
```diff
@@ -48,13 +76,15 @@ impl Environment for CustomEnvironment {
 #[ink::contract(env = crate::CustomEnvironment)]
 mod dapp_staking_extension {
     use super::CurrentEraErr;
+    use super::EraInfo;
     /// Defines the storage of our contract.
     ///
     /// Here we store the random seed fetched from the chain.
```
Comment (PierreOssun) [inline dapps-staking-chain-extension/lib.rs]: Description doesn't match I guess

File: dapps-staking-chain-extension/lib.rs [hunk 2]
```diff
@@ -67,7 +97,7 @@ mod dapp_staking_extension {
         /// Constructor that initializes the `bool` value to the given `init_value`.
         #[ink(constructor)]
         pub fn new(init_value: u32) -> Self {
-            Self { value: init_value }
+            Self { value: init_value, staked: 100}
```
Comment (PierreOssun) [inline dapps-staking-chain-extension/lib.rs]: better `staked: 0` ?

File: dapps-staking-chain-extension/lib.rs [hunk 3]
```diff
@@ -95,6 +125,17 @@ mod dapp_staking_extension {
         pub fn get_current_era(&self) -> u32 {
             self.value
         }
+
+        /// Calls current_era() in the pallet-dapps-staking
+        #[ink(message)]
+        pub fn read_era_info(&mut self, era:u32) -> Result<(), CurrentEraErr> {
```
Comment (PierreOssun) [inline dapps-staking-chain-extension/lib.rs]: This fun modify the state so it should not be called `read`

File: dapps-staking-chain-extension/lib.rs [hunk 4]
```diff
@@ -95,6 +125,17 @@ mod dapp_staking_extension {
...
+
```
Comment (PierreOssun) [inline dapps-staking-chain-extension/lib.rs]: blank line

**Pull request merged**